### PR TITLE
Feature/#8

### LINKED ISSUE #8: create PostgreSQL database in 'db' folder
folder contains:

1. Create folder 'data' (contains SQL files for fishing/filling data from database).
2. Create folder 'schema' (contains SQL files for creating database).
3. Create 'db.sql' file.
4. Create 'insert_data.sql' file.
5. Add block for init database to `Readme.md`.

### PR DESCRIPTION
**Pull request by lurak**
Created folder 'data' (contains SQL files for fishing/filling data from database).
Created folder 'schema' (contains SQL files for creating database).
Created 'db.sql' file.
Created 'insert_data.sql' file.
Added block for init database to Readme.md.

### CODE REVIEW

**Review by daniiloleshchuk — APPROVED** [review]

**Review by SLDem — APPROVED** [review]

**Review by StepanTchynetskyi — APPROVED** [review]

File: db/StockDashboard/schema/dashboard_config.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS dashboard_config(
+    id serial PRIMARY KEY,
+    hash TEXT NOT NULL
```
Comment (lhalam) [inline db/StockDashboard/schema/dashboard_config.sql]: change to VARCHAR(8)

File: db/StockDashboard/schema/stocks.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS stocks(
+    id serial PRIMARY KEY,
+    name TEXT NOT NULL,
```
Comment (lhalam) [inline db/StockDashboard/schema/stocks.sql]: change to VARCHAR(16)

File: db/StockDashboard/schema/stocks.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    company_name TEXT NOT NULL
```
Comment (lhalam) [inline db/StockDashboard/schema/stocks.sql]: change to VARCHAR(128)

File: db/StockDashboard/schema/stocks_data.sql [hunk 4]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS stocks_data(
+    id serial PRIMARY KEY,
+    stock_id INTEGER NOT NULL ,
+    price FLOAT NOT NULL,
+    date_time TIMESTAMP NOT NULL
```
Comment (lhalam) [inline db/StockDashboard/schema/stocks_data.sql]: rename to `create_at`

File: db/StockDashboard/data/stocks_data.sql [hunk 5]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO stocks_data(stock_id, price, date_time) VALUES
+    (1, 144.15, '2020-04-01'),
```
Comment (lhalam) [inline db/StockDashboard/data/stocks_data.sql]: add  HH:MI

**Pull request merged**